indigo_app/templates/indigo_api/work_overview.html

```diff
@@ -19,6 +19,7 @@ <h3>
     {% if not work.commenced %} <span class="badge badge-info">not commenced</span>
     {% elif not work.commencement_date %} <span class="badge badge-info">commencement date unknown</span>
     {% endif %}
+    {% if work.as_at_date %}<span class="badge badge-info" title="{{ work.as_at_date|date:'Y-m-d' }}">as-at-overridden</span>{% endif %}
```

Comment:
I think this looks a bit clumsy.

How about putting it here:

![image](https://user-images.githubusercontent.com/4178542/125962540-a1022708-9fb2-4b3a-a894-58ab4c26b132.png)

```
<whatever the heading tag is>Up-to-date as at</...>

{{ calculated as at date }}
{% if work.as_at_date_override %}
<span class="text-muted" title="Set explicitly on this work"><i class="fa fa-info-circle"></i> Overidden</span>
{% endif %}